ui/app/AppLayouts/Wallet/panels/SwapInputPanel.qml

```diff
@@ -63,6 +63,10 @@ Control {
     property int swapExchangeButtonWidth: 44
     property string caption: swapSide === SwapInputPanel.SwapSide.Pay ? qsTr("Pay") : qsTr("Receive")
 
+    function forceActiveFocus() {
```

Comment:
We're already doing this below:
```qml
    Component.onCompleted: {
        if (root.swapSide === SwapInputPanel.SwapSide.Pay)
            amountToSendInput.input.forceActiveFocus()
    }
```

I wonder why it doesn't work for you... :thinking:

Reply:
Hey this is for after we press the exchange button, after that the focus would lie with the button and not the input right. thats why its needed. I dont see the cursor still, but that a separate bug

Reply:
Ah, then set the correct focus policy on the exchange button, in order not to steal it

Reply:
no but there is some logic involved here. after the clicking swap the focus should move the the token which has focus. so in case pay side had focus and we click exchange button it should move to receive side ... @caybro